spec.html

```diff
@@ -7629,11 +7607,19 @@ <h1>EnqueueJob ( _queueName_, _job_, _arguments_ )</h1>
         1. Let _callerRealm_ be _callerContext_'s Realm.
         1. Let _callerScriptOrModule_ be _callerContext_'s ScriptOrModule.
         1. Let _pending_ be PendingJob { [[Job]]: _job_, [[Arguments]]: _arguments_, [[Realm]]: _callerRealm_, [[ScriptOrModule]]: _callerScriptOrModule_, [[HostDefined]]: *undefined* }.
-        1. Perform any implementation or host environment defined processing of _pending_. This may include modifying the [[HostDefined]] field or any other field of _pending_.
-        1. Add _pending_ at the back of the Job Queue named by _queueName_.
+        1. Let _result_ be the result of performing the abstract operation named by _hostHook_ using &laquo; _pending_ &raquo; as its arguments.
```

Comment:
```suggestion
        1. Let _result_ be the result of performing the abstract operation referenced by _hostHook_ using &laquo; _pending_ &raquo; as its arguments.
```
? it's not a name anymore, now it seems like a first-class reference.

Reply:
Don't names reference things? I can change this back if you prefer, but I don't see why.

Reply:
Err, I misread @ljharb 's comment. I like this suggestion and am committing it.

Reply:
Ugh, the GitHub UI won't let me commit it. I'll make this change together with whatever other editor review changes come next, or if this is ready to merge without changes, I'm happy with the editors merging in the change.